plugins/dapper/README.md

```diff
@@ -0,0 +1,168 @@
+# Dapper Plugin for SBOM Surfactant
+
+A plugin for Surfactant that uses [Dapper](https://github.com/LLNL/dapper) to identify software packages from binary files by querying comprehensive package databases.
+
+## Overview
+
+The Dapper plugin enhances Surfactant's SBOM generation by identifying which software packages provide specific binary files. It queries multiple Linux distribution databases (Ubuntu, Debian) to find package information for ELF files, with planned support for Windows PE files via NuGet databases.
+
+## Features
+
+- **Multi-distribution package detection**: Queries all available Linux distributions (Ubuntu Focal/Jammy/Noble, Debian Buster/Bullseye/Bookworm)
+- **Comprehensive results**: Returns all matching packages across distributions
+- **File normalization**: Handles various file naming conventions through intelligent normalization
+- **Structured output**: Provides detailed package information including package names, versions, and source distributions
+
+## Prerequisites
+
+### Install Dapper and Datasets
+
+Before using this plugin, you need to install Dapper and download the package databases:
+
+1. Install dapper-python (required for database queries):
+   ```bash
+   pip install dapper-python
+   ```
+
+2. Clone and build Dapper:
+   ```bash
+   git clone https://github.com/LLNL/dapper.git
+   cd dapper
+   cargo build --release
+   ```
+
+3. Download required datasets (must be run from the dapper directory):
+   ```bash
+   # List available datasets
+   cargo run -- . --list-available-datasets
+
+   # Install specific Linux datasets
+   cargo run -- . --install ubuntu-focal
+   cargo run -- . --install ubuntu-jammy
+   cargo run -- . --install ubuntu-noble
+   cargo run -- . --install debian-bookworm
+
+   # Install all available datasets
+   cargo run -- . --install all
+   ```
```

Comment:
This can be updated later to use `cargo install dapper`, so no cloning is required, and then the user will have a binary on their path that lets them use `dapper` instead of `cargo run --` in all the subsequent commands for installing datasets.